decent-dao/fractal-interface
# Handle multiSend for Usul proposals

## Description
This PR adjusts data decoding for `Usul`-based proposals and handles unpacking "internal" transactions from `multiSend`.

## Notes

N/A

## Issue / Notion doc (if applicable)

Closes #787 

## Testing

1. Create subDAO on Usul-based parent DAO
2. Verify that expanded proposal details are shown properly
3. Alternatively - there's existing DAO with such a proposal `daos/0xF332aAb56e00f7FDEdEDDC50033Ce5e2D0DD6d61/proposals/1`
4. Alternatively - just create proposal with `multiSend` execution (you'll have to decode data yourself)

## Screenshots (if applicable)
<img width="1680" alt="Знімок екрана 2023-01-03 о 22 32 37" src="https://user-images.githubusercontent.com/25801162/210437315-664cca80-d849-4b4c-bbdd-d0c5954bd830.png">
